### ISSUE TITLE
pkgdown site needed

### ISSUE THREAD

**Issue by florisvdh** [inbo/inbospatial#10]
I think **checklist** also provides this.

**Comment by hansvancalster** [comment]
Yes, I know and noticed this too. I asked @ThierryO to see what is the matter.

**Comment by florisvdh** [comment]
Alternatively you can do [`usethis::use_pkgdown_github_pages()`](https://usethis.r-lib.org/reference/use_pkgdown.html).

**Comment by ThierryO** [comment]
I just forced push an updated `release.yml`

**Comment by ThierryO** [comment]
Done. The gh-pages website wasn't activated in the settings.
